package.json

```diff
@@ -1,8 +1,69 @@
-{
-  "devDependencies": {
-    "file-system": "^2.2.2",
-    "jsdoc-to-markdown": "^6.0.1",
-    "limax": "^2.1.0"
-  },
-  "dependencies": {}
-}
+{
+  "name": "root",
+  "private": true,
+  "workspaces": [
+    "packages/*"
+  ],
+  "scripts": {
+    "generate": "node scripts/generate/generate.mjs",
+    "clean": "rimraf packages/**/dist packages/**/node_modules/.cache",
+    "docs": "typedoc",
+    "lint": "eslint packages --ext mjs,ts --fix",
+    "format": "prettier --write {packages,scripts,tests}/**/*.{mjs,ts}",
+    "test": "jest --verbose",
+    "build": "yarn workspaces run build",
+    "prepublishOnly": "yarn workspaces run prepublishOnly",
+    "update": "yarn upgrade-interactive --latest",
+    "prepare": "husky install"
+  },
+  "devDependencies": {
+    "@commitlint/cli": "^16.1.0",
+    "@commitlint/config-conventional": "^16.0.0",
+    "@favware/rollup-type-bundler": "^1.0.5",
+    "@sapphire/eslint-config": "^4.0.11",
+    "@sapphire/ts-config": "^3.1.8",
+    "@types/jest": "^27.0.2",
+    "@types/node": "16",
+    "@types/prompts": "^2.0.14",
+    "@types/uuid": "^8.3.1",
+    "@typescript-eslint/eslint-plugin": "^5.10.1",
+    "colorette": "^2.0.16",
+    "eslint": "~8.6.0",
+    "eslint-config-prettier": "^8.3.0",
+    "husky": "^7.0.4",
+    "jest": "^27.2.5",
+    "jest-circus": "^27.2.5",
+    "lint-staged": "^12.3.2",
+    "ora": "^6.0.1",
+    "prettier": "^2.4.1",
+    "pretty-quick": "^3.1.1",
+    "prompts": "^2.4.2",
+    "rollup": "^2.66.1",
+    "rollup-plugin-cleaner": "^1.0.0",
+    "rollup-plugin-typescript2": "^0.31.1",
+    "rollup-plugin-version-injector": "^1.3.3",
+    "ts-jest": "^27.1.3",
+    "ts-node": "^10.2.1",
+    "typedoc": "^0.22.11",
+    "typescript": "^4.5.5"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/josh-development/providers.git"
+  },
+  "engines": {
+    "node": ">=16",
+    "npm": ">=6"
```

Comment:
It's not necessary since you'd be using Yarn v1 no matter what. The only way you can accidently use any other version is buying creating files in the workspace specifying the version of Yarn.